packages/react-components/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -81,6 +86,7 @@ export const ActionMenu: React.FC<ActionMenuProps> = ({
   visible,
   onClose,
   onOpen,
+  floatingStrategy = 'absolute',
```

Comment:
there is no need to set the fault value here - it already has this value by the library side